main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+func main() {
+	logger := utils.NewLogger(&utils.LoggerConfig{
+		Environment: "development",
+		Path:        "./test.log",
+	})
+
+	url := "hello"
+
+	logger.Error(fmt.Sprintf("string %s", url))
+}
```

Comment:
An [example](https://blog.golang.org/examples) might be better suited than a dedicated main here.